### PR TITLE
Fix newly found issues due to latest changes

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Fix circular node module loading issue
- Fix start node script argument number issue 

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1216
- https://github.com/ainblockchain/ain-blockchain/issues/1217

### CODE REVIEW

File: start_node_genesis_gcp.sh [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 # NOTE(minsulee2): Since exit really exits terminals, those are replaced to return 1.
-if [[ $# -lt 4 ]] || [[ $# -gt 12 ]]; then
-    printf "Usage: bash start_node_genesis_gcp.sh [dev|staging|sandbox|exp|spring|summer|mainnet] <GCP Username> <Shard Index> <Node Index> [--keystore|--mnemonic|--private-key] [--keep-code|--no-keep-code] [--keep-data|--no-keep-data] [--full-sync|--fast-sync] [--chown-data|--no-chown-data] [--json-rpc] [--update-front-db] [--rest-func]\n"
+if [[ $# -lt 4 ]] || [[ $# -gt 13 ]]; then
```
Comment (jiyoung-an) [inline start_node_genesis_gcp.sh:4]: what is the meaning of number `13` and `4`?

**Review by jiyoung-an — APPROVED** [review]
LGTM with a question! Thanks!

File: start_node_genesis_gcp.sh (hunk 1 above)
Comment (platfowner) [reply start_node_genesis_gcp.sh:4]: That's the maximum number of input parameters in sh.

**Comment by platfowner** [comment]
Thanks for the review! @jiyoung-an Now merging..

**Pull request merged**